docs/modules/ROOT/pages/documentation/ecosystem/arquillian-containers/payara-micro-managed.adoc

```diff
@@ -90,17 +90,15 @@ To change this, provide java debug options to `cmdOptions`.
 |===
 
 These options can all be provided through either an environment variable or system property.
-The easiest way to provide these to the test instance is through the surefire plugin.
+The easiest way to provide these to the test instance is is by using an `arquillian.xml` file.
 
 [source,XML]
 .Example
 ----
-<plugin>
-    <artifactId>maven-surefire-plugin</artifactId>
+<container qualifier="payara-micro-managed">
```

Comment:
I'd add a full arquillian.xml example to help people create it without searching the Arquillian docs:

```suggestion
<arquillian xmlns="http://jboss.org/schema/arquillian"
    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
    xsi:schemaLocation="
        http://jboss.org/schema/arquillian
        http://jboss.org/schema/arquillian/arquillian_1_0.xsd">
<container qualifier="payara-micro-managed">
```